corehq/form_processor/interfaces/ledger_processor.py

```diff
@@ -17,10 +17,10 @@ def __new__(cls, to_save=None, to_delete=None):
         )
 
     def commit(self):
```

Comment:
not 100% sure it does but there are a bunch of signal handers on StockTransaction events that make me nervous: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/stock/signals.py#L11-L25